code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -41,6 +42,8 @@
 	vis_contents += occupant
 	pixel_y = 22
 	ADD_TRAIT(occupant, TRAIT_IMMOBILIZED, CRYO_TRAIT)
+	// Keep them standing! They'll go sideways in the tube when they fall asleep otherwise.
+	ADD_TRAIT(occupant, TRAIT_FORCED_STANDING, CRYO_TRAIT)
 	occupant.set_body_position(STANDING_UP)
 	occupant.set_lying_angle(0)
```

Comment:
```suggestion
	ADD_TRAIT(occupant, TRAIT_FORCED_STANDING, CRYO_TRAIT)
```
These are now already processed when the trait is added for the first time.

Reply:
Done!